Common/Orders/Fees/BinanceFeeModel.cs

```diff
@@ -0,0 +1,86 @@
+﻿/*
+* QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+* Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using QuantConnect.Securities;
+
+namespace QuantConnect.Orders.Fees
+{
+    /// <summary>
+    /// Provides an implementation of <see cref="FeeModel"/> that models Binance order fees
+    /// </summary>
+    public class BinanceFeeModel : FeeModel
+    {
+        /// <summary>
+        /// Tier 1 maker fees
+        /// https://www.binance.com/en/fee/schedule
+        /// </summary>
+        public const decimal MakerTear1Fee = 0.001m;
+        /// <summary>
+        /// Tier 1 taker fees
+        /// https://www.binance.com/en/fee/schedule
+        /// </summary>
+        public const decimal TakerTear1Fee = 0.001m;
+
+        private readonly decimal makerFee;
+        private readonly decimal takerFee;
```

Comment:
nit - To follow code style should `_`
Seems very similar to `BitfinexFeeModel` think there is room for removing some code duplication, the only difference is determining if should use taker or market fee